mxcube3/routes/Utils.py

```diff
@@ -95,12 +97,142 @@ def my_execute_entry(self, entry):
         foo = ['collectOscillationFinished', 'collectOscillationFailed', 'warning']
         mxcube.collect.emit(random.choice(foo))
     elif isinstance(entry, qe.CharacterisationGroupQueueEntry):
+        logging.getLogger('HWR').info('[QUEUE] Executing CharacterisationGroupQueueEntry entry: %s' %entry)
```

Comment:
I started this pr before your big refactoring, so looking commit by commit you could see that, but it was not in the final code.